### PR TITLE
Add polygon to python ROIs examples

### PR DESCRIPTION
**Pull request by will-moore**
This updates the Python ROIs example code according to changes in the examples at https://github.com/openmicroscopy/openmicroscopy/pull/4412 (Polygons)
and 
https://github.com/openmicroscopy/openmicroscopy/pull/4415 (Masks).
Check for formatting errors etc.
Code changes themselves are checked by the above PRs.

### CODE REVIEW

**Comment by will-moore** [comment]
Probably best to wait for the code examples PR above to be merged, so code changes can be updated in the docs too.

**Comment by will-moore** [comment]
@sbesson Can you remove the ON_HOLD label now? Thanks.

**Comment by joshmoore** [comment]
Done. Worth integrating my masks changes here as well (assuming they make it in tomorrow)?

**Comment by will-moore** [comment]
@joshmoore Yes, I was just thinking that.

File: omero/developers/Python.txt [hunk 1]
```diff
@@ -781,12 +781,6 @@ ROIs
     conn.connect()
     updateService = conn.getUpdateService()
 
--  **Configuration**
-
-::
-
-    imageId = 27544
```
Comment (ximenesuk) [inline omero/developers/Python.txt:788]: Isn't an `imageId` still needed?

File: omero/developers/Python.txt [hunk 2]
```diff
@@ -839,13 +845,17 @@ ROIs
...
     ellipse.textValue = rstring("test-Ellipse")
-    roi.addShape(ellipse) 
 
 ::
 
-    # create a line shape and add to ROI
-    print ("Adding line at theZ: %s, theT: %s, x1: %s, y1: %s, x2: %s,"
-       " y2: %s" % (theZ, theT, x, y, x+width, y+height))
+    # Create an ROI containing 2 shapes on same plane
+    # NB: OMERO.insight client doesn't support this
```
Comment (ximenesuk) [inline omero/developers/Python.txt]: Doesn't support viewing, creation?

File: omero/developers/Python.txt [hunk 3]
```diff
@@ -854,23 +864,104 @@ ROIs
...
-::
-
-    # Save the ROI (saves any linked shapes too)
-    r = updateService.saveAndReturnObject(roi) 
+    createROI(image, [point])
+
+::
+
+    def pointsToString(points):
+        """ Returns strange format supported by Insight """
```
Comment (ximenesuk) [inline omero/developers/Python.txt]: strange? That's a bit loaded, could the comment be expressed a little better?

**Comment by ximenesuk** [comment]
The workflow looks okay but I have not explicitly tested the code.

File: omero/developers/Python.txt (hunk 2 above)
Comment (will-moore) [inline omero/developers/Python.txt]: Neither viewing nor creation

File: omero/developers/Python.txt (hunk 3 above)
Comment (will-moore) [inline omero/developers/Python.txt]: I'm open to suggestions!
Maybe I should also check if a more sensible format is supported by Insight & web.
Any changes here need to be ported back to a new PR to update examples.

File: omero/developers/Python.txt (hunk 2 above)
Comment (ximenesuk) [inline omero/developers/Python.txt]: Could the comment be expanded to reflect that?

**Comment by joshmoore** [comment]
Code should have been tested in the related PRs.

File: omero/developers/Python.txt (hunk 1 above)
Comment (ximenesuk) [inline omero/developers/Python.txt:788]: Scratch this comment, I looked at the change in isolation rather then checking the whole file.

**Comment by ximenesuk** [comment]
Thanks @joshmoore for some reason I completely failed to see the description, just jumping to the staging docs and the diffs!

File: omero/developers/Python.txt (hunk 3 above)
Comment (joshmoore) [inline omero/developers/Python.txt]: You could call it the "legacy" format.

Comment (ximenesuk) [inline omero/developers/Python.txt]: That would work for me.

**Comment by hflynn** [comment]
Code PRs merged, is this good to merge too @ximenesuk @joshmoore ?

**Comment by ximenesuk** [comment]
@hflynn I'd not seen the final commit but that looks fine so I'm happy for it to be merged.

**Comment by joshmoore** [comment]
Yup, merging, but he change will need to be backported to https://github.com/openmicroscopy/openmicroscopy/pull/4412/files#diff-721941e13150907e2a501f3e5cef6ad3R115 otherwise it'll propagate back to the documentation.

**Comment by hflynn** [comment]
@will-moore leaving you to deal with the backporting Josh mentioned.

**Pull request merged**